lib/ui/profitloss/RecordAmount.dart

```diff
@@ -0,0 +1,295 @@
+import 'package:farmsmart_flutter/model/loading_status.dart';
+import 'package:farmsmart_flutter/ui/common/ListDivider.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:farmsmart_flutter/ui/mockData/MockRecordAmountViewModel.dart';
+import 'package:farmsmart_flutter/ui/mockData/MockRoundedButtonViewModel.dart';
+import 'package:farmsmart_flutter/ui/profitloss/RecordAmountListItem.dart';
+import 'package:farmsmart_flutter/ui/profitloss/RecordAmountHeader.dart';
+import 'package:farmsmart_flutter/ui/profitloss/RecordAmountListItem.dart';
+import 'package:farmsmart_flutter/ui/profitloss/RecordAmountListItemStyles.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:flutter/material.dart';
+
+enum RecordType {
+  cost,
+  sale,
+}
+
+class RecordAmountViewModel {
+  LoadingStatus loadingStatus;
+  List<RecordAmountListItemViewModel> actions;
+  String amount;
+  String buttonTitle;
+  bool isFilled;
+  Function onTap;
+  RecordType type;
+  bool isEditable;
+  String description;
+
+  RecordAmountViewModel(
+      {this.loadingStatus,
+      this.actions,
+      this.amount: "00",
+      this.buttonTitle,
+      this.isFilled: false,
+      this.onTap,
+      this.type,
+      this.isEditable: true});
```

Comment:
```suggestion
      this.isEditable: true,});
```
After change, format code

Reply:
Done